MIGRATING.md

```diff
@@ -0,0 +1,214 @@
+# Migration Guide
+
+When we update `@fastly/compute-js-static-publish` to a new major version, sometimes we make changes to the files that
+are generated during scaffolding. For this reason, if possible we recommended that you re-scaffold your application.
+
+This is pretty simple if you've been using your `compute-js-static-publisher` application without modification, but we
+know that's not always possible, so the following is provided to help you overcome any breaking changes.
+
+# Migrating to v4
+
+## Webpack
+
+Starting with `v4.0.0` of this tool, Webpack is no longer required, and is disabled by default for new applications
+scaffolded using it. This can simplify your development and give you quicker build times. However, you may still wish
+to use Webpack if you need some of the features it provides, such as the ability to use loaders, asset modules, module
+replacement, dynamic imports, etc.
```

Comment:
```suggestion
Starting with `v4.0.0` of this tool, Webpack is no longer required and is disabled by default for new applications. This can simplify development and result in shorter build times. 

You may still wish to use Webpack if you need some of the features it provides, e.g., the ability to use loaders, asset modules, module replacement, dynamic imports, etc.
```